<?xml version="1.0" encoding="UTF-8" ?>
<testsuite failures="33" time="82.913" errors="0" skipped="0" tests="135" name="TestSuite">
  <properties>
    <property name="java.runtime.name" value="Java(TM) SE Runtime Environment"/>
    <property name="sun.boot.library.path" value="C:\Program Files\Java\jdk1.7.0_80\jre\bin"/>
    <property name="java.vm.version" value="24.80-b11"/>
    <property name="java.vm.vendor" value="Oracle Corporation"/>
    <property name="java.vendor.url" value="http://java.oracle.com/"/>
    <property name="path.separator" value=";"/>
    <property name="java.vm.name" value="Java HotSpot(TM) 64-Bit Server VM"/>
    <property name="file.encoding.pkg" value="sun.io"/>
    <property name="user.script" value=""/>
    <property name="user.country" value="CN"/>
    <property name="sun.java.launcher" value="SUN_STANDARD"/>
    <property name="sun.os.patch.level" value="Service Pack 1"/>
    <property name="java.vm.specification.name" value="Java Virtual Machine Specification"/>
    <property name="user.dir" value="E:\java_code\tws2.0_maven"/>
    <property name="java.runtime.version" value="1.7.0_80-b15"/>
    <property name="java.awt.graphicsenv" value="sun.awt.Win32GraphicsEnvironment"/>
    <property name="basedir" value="E:\java_code\tws2.0_maven"/>
    <property name="java.endorsed.dirs" value="C:\Program Files\Java\jdk1.7.0_80\jre\lib\endorsed"/>
    <property name="os.arch" value="amd64"/>
    <property name="surefire.real.class.path" value="C:\Users\tangtao\AppData\Local\Temp\surefirebooter8749519873656682686.jar"/>
    <property name="java.io.tmpdir" value="C:\Users\tangtao\AppData\Local\Temp\"/>
    <property name="line.separator" value="
"/>
    <property name="java.vm.specification.vendor" value="Oracle Corporation"/>
    <property name="user.variant" value=""/>
    <property name="os.name" value="Windows 7"/>
    <property name="sun.jnu.encoding" value="GBK"/>
    <property name="java.library.path" value="C:\Program Files\Java\jdk1.7.0_80\jre\bin;C:\Windows\Sun\Java\bin;C:\Windows\system32;C:\Windows;C:\apache-maven-3.3.9/bin;C:\Program Files\Java\jdk1.7.0_80/bin;C:\Program Files\Java\jdk1.7.0_80/bin;C:\Python27\Lib\site-packages\PyQt4;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Program Files (x86)\ATI Technologies\ATI.ACE\Core-Static;C:\Python27;C:\Python27\Lib\site-packages;C:\Program Files\Java\jdk1.7.0_80;C:\Program Files\TortoiseSVN\bin;C:\Program Files (x86)\Windows Kits\8.1\Windows Performance Toolkit\;C:\Program Files\Microsoft SQL Server\110\Tools\Binn\;C:\Python27\Scripts;C:\Program Files (x86)\Google\Chrome;C:\Program Files (x86)\nodejs\;C:\Program Files (x86)\Appium;C:\Users\tangtao\AppData\Local\Android\android-sdk\platform-tools;C:\Users\tangtao\AppData\Local\Android\android-sdk\tools;C:\Program Files\Java\jdk1.7.0_80\bin;C:\Program Files\Java\jdk1.7.0_80\jre\bin;C:\Program Files (x86)\Appium\node_modules\.bin;C:\Program Files\MySQL\MySQL Server 5.5\bin;C:\apache-maven-3.3.9\bin;C:\Program Files (x86)\Android\android-sdk\platform-tools;C:\apache-tomcat-7.0.64\lib;C:\apache-tomcat-7.0.64\bin;C:\Users\tangtao\AppData\Roaming\npm;."/>
    <property name="surefire.test.class.path" value="E:\java_code\tws2.0_maven\target\test-classes;E:\java_code\tws2.0_maven\target\classes;C:\Users\tangtao\.m2\repository\org\apache\poi\poi-ooxml\3.13\poi-ooxml-3.13.jar;C:\Users\tangtao\.m2\repository\org\apache\poi\poi\3.13\poi-3.13.jar;C:\Users\tangtao\.m2\repository\org\apache\poi\poi-ooxml-schemas\3.13\poi-ooxml-schemas-3.13.jar;C:\Users\tangtao\.m2\repository\org\apache\xmlbeans\xmlbeans\2.6.0\xmlbeans-2.6.0.jar;C:\Users\tangtao\.m2\repository\stax\stax-api\1.0.1\stax-api-1.0.1.jar;C:\Users\tangtao\.m2\repository\org\testng\testng\6.9.10\testng-6.9.10.jar;C:\Users\tangtao\.m2\repository\com\beust\jcommander\1.48\jcommander-1.48.jar;C:\Users\tangtao\.m2\repository\org\beanshell\bsh\2.0b4\bsh-2.0b4.jar;C:\Users\tangtao\.m2\repository\org\apache\httpcomponents\httpclient\4.5.2\httpclient-4.5.2.jar;C:\Users\tangtao\.m2\repository\org\apache\httpcomponents\httpcore\4.4.4\httpcore-4.4.4.jar;C:\Users\tangtao\.m2\repository\commons-logging\commons-logging\1.2\commons-logging-1.2.jar;C:\Users\tangtao\.m2\repository\commons-codec\commons-codec\1.9\commons-codec-1.9.jar;C:\Users\tangtao\.m2\repository\net\sf\json-lib\json-lib\2.4\json-lib-2.4-jdk15.jar;C:\Users\tangtao\.m2\repository\commons-beanutils\commons-beanutils\1.8.0\commons-beanutils-1.8.0.jar;C:\Users\tangtao\.m2\repository\commons-collections\commons-collections\3.2.1\commons-collections-3.2.1.jar;C:\Users\tangtao\.m2\repository\commons-lang\commons-lang\2.5\commons-lang-2.5.jar;C:\Users\tangtao\.m2\repository\net\sf\ezmorph\ezmorph\1.0.6\ezmorph-1.0.6.jar;C:\Users\tangtao\.m2\repository\mysql\mysql-connector-java\5.1.38\mysql-connector-java-5.1.38.jar;C:\Users\tangtao\.m2\repository\org\uncommons\reportng\1.1.2\reportng-1.1.2.jar;C:\Users\tangtao\.m2\repository\velocity\velocity\1.4\velocity-1.4.jar;C:\Users\tangtao\.m2\repository\velocity\velocity-dep\1.4\velocity-dep-1.4.jar;C:\Users\tangtao\.m2\repository\com\google\inject\guice\3.0\guice-3.0.jar;C:\Users\tangtao\.m2\repository\javax\inject\javax.inject\1\javax.inject-1.jar;C:\Users\tangtao\.m2\repository\aopalliance\aopalliance\1.0\aopalliance-1.0.jar;C:\Users\tangtao\.m2\repository\com\google\code\gson\gson\2.3\gson-2.3.jar;"/>
    <property name="java.specification.name" value="Java Platform API Specification"/>
    <property name="java.class.version" value="51.0"/>
    <property name="sun.management.compiler" value="HotSpot 64-Bit Tiered Compilers"/>
    <property name="os.version" value="6.1"/>
    <property name="user.home" value="C:\Users\tangtao"/>
    <property name="user.timezone" value="Asia/Shanghai"/>
    <property name="java.awt.printerjob" value="sun.awt.windows.WPrinterJob"/>
    <property name="java.specification.version" value="1.7"/>
    <property name="file.encoding" value="GBK"/>
    <property name="user.name" value="tangtao"/>
    <property name="java.class.path" value="E:\java_code\tws2.0_maven\target\test-classes;E:\java_code\tws2.0_maven\target\classes;C:\Users\tangtao\.m2\repository\org\apache\poi\poi-ooxml\3.13\poi-ooxml-3.13.jar;C:\Users\tangtao\.m2\repository\org\apache\poi\poi\3.13\poi-3.13.jar;C:\Users\tangtao\.m2\repository\org\apache\poi\poi-ooxml-schemas\3.13\poi-ooxml-schemas-3.13.jar;C:\Users\tangtao\.m2\repository\org\apache\xmlbeans\xmlbeans\2.6.0\xmlbeans-2.6.0.jar;C:\Users\tangtao\.m2\repository\stax\stax-api\1.0.1\stax-api-1.0.1.jar;C:\Users\tangtao\.m2\repository\org\testng\testng\6.9.10\testng-6.9.10.jar;C:\Users\tangtao\.m2\repository\com\beust\jcommander\1.48\jcommander-1.48.jar;C:\Users\tangtao\.m2\repository\org\beanshell\bsh\2.0b4\bsh-2.0b4.jar;C:\Users\tangtao\.m2\repository\org\apache\httpcomponents\httpclient\4.5.2\httpclient-4.5.2.jar;C:\Users\tangtao\.m2\repository\org\apache\httpcomponents\httpcore\4.4.4\httpcore-4.4.4.jar;C:\Users\tangtao\.m2\repository\commons-logging\commons-logging\1.2\commons-logging-1.2.jar;C:\Users\tangtao\.m2\repository\commons-codec\commons-codec\1.9\commons-codec-1.9.jar;C:\Users\tangtao\.m2\repository\net\sf\json-lib\json-lib\2.4\json-lib-2.4-jdk15.jar;C:\Users\tangtao\.m2\repository\commons-beanutils\commons-beanutils\1.8.0\commons-beanutils-1.8.0.jar;C:\Users\tangtao\.m2\repository\commons-collections\commons-collections\3.2.1\commons-collections-3.2.1.jar;C:\Users\tangtao\.m2\repository\commons-lang\commons-lang\2.5\commons-lang-2.5.jar;C:\Users\tangtao\.m2\repository\net\sf\ezmorph\ezmorph\1.0.6\ezmorph-1.0.6.jar;C:\Users\tangtao\.m2\repository\mysql\mysql-connector-java\5.1.38\mysql-connector-java-5.1.38.jar;C:\Users\tangtao\.m2\repository\org\uncommons\reportng\1.1.2\reportng-1.1.2.jar;C:\Users\tangtao\.m2\repository\velocity\velocity\1.4\velocity-1.4.jar;C:\Users\tangtao\.m2\repository\velocity\velocity-dep\1.4\velocity-dep-1.4.jar;C:\Users\tangtao\.m2\repository\com\google\inject\guice\3.0\guice-3.0.jar;C:\Users\tangtao\.m2\repository\javax\inject\javax.inject\1\javax.inject-1.jar;C:\Users\tangtao\.m2\repository\aopalliance\aopalliance\1.0\aopalliance-1.0.jar;C:\Users\tangtao\.m2\repository\com\google\code\gson\gson\2.3\gson-2.3.jar;"/>
    <property name="java.vm.specification.version" value="1.7"/>
    <property name="sun.arch.data.model" value="64"/>
    <property name="java.home" value="C:\Program Files\Java\jdk1.7.0_80\jre"/>
    <property name="sun.java.command" value="C:\Users\tangtao\AppData\Local\Temp\surefirebooter8749519873656682686.jar C:\Users\tangtao\AppData\Local\Temp\surefire8149772253740743244tmp C:\Users\tangtao\AppData\Local\Temp\surefire1966316323702555844tmp"/>
    <property name="java.specification.vendor" value="Oracle Corporation"/>
    <property name="user.language" value="zh"/>
    <property name="awt.toolkit" value="sun.awt.windows.WToolkit"/>
    <property name="java.vm.info" value="mixed mode"/>
    <property name="java.version" value="1.7.0_80"/>
    <property name="java.ext.dirs" value="C:\Program Files\Java\jdk1.7.0_80\jre\lib\ext;C:\Windows\Sun\Java\lib\ext"/>
    <property name="sun.boot.class.path" value="C:\Program Files\Java\jdk1.7.0_80\jre\lib\resources.jar;C:\Program Files\Java\jdk1.7.0_80\jre\lib\rt.jar;C:\Program Files\Java\jdk1.7.0_80\jre\lib\sunrsasign.jar;C:\Program Files\Java\jdk1.7.0_80\jre\lib\jsse.jar;C:\Program Files\Java\jdk1.7.0_80\jre\lib\jce.jar;C:\Program Files\Java\jdk1.7.0_80\jre\lib\charsets.jar;C:\Program Files\Java\jdk1.7.0_80\jre\lib\jfr.jar;C:\Program Files\Java\jdk1.7.0_80\jre\classes"/>
    <property name="java.vendor" value="Oracle Corporation"/>
    <property name="localRepository" value="C:\Users\tangtao\.m2\repository"/>
    <property name="file.separator" value="\"/>
    <property name="java.vendor.url.bug" value="http://bugreport.sun.com/bugreport/"/>
    <property name="sun.cpu.endian" value="little"/>
    <property name="sun.io.unicode.encoding" value="UnicodeLittle"/>
    <property name="sun.desktop" value="windows"/>
    <property name="sun.cpu.isalist" value="amd64"/>
  </properties>
  <testcase time="1.675" classname="main.testcase.RegisterAndlogin.login" name="login"/>
  <testcase time="0.884" classname="main.testcase.RegisterAndlogin.login" name="login"/>
  <testcase time="1.014" classname="main.testcase.RegisterAndlogin.login" name="login">
    <failure message="expected [空] but found []" type="java.lang.AssertionError">java.lang.AssertionError: expected [空] but found []
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at main.testcase.RegisterAndlogin.login.login(login.java:65)
</failure>
    <system-out>{&quot;systemVersion&quot;:&quot;1&quot;,&quot;appVersion&quot;:&quot;1&quot;,&quot;clientIP&quot;:&quot;192.168.9.55&quot;,&quot;deviceNumber&quot;:&quot;a750a6d6-edf1-4704-9146-bc7eed651d65&quot;,&quot;sessionGuid&quot;:&quot;b43c796e-1b43-45e8-a2bb-d84dd01ad034&quot;,&quot;token&quot;:&quot;4ab31229-ce88-4b70-8aa6-46f2c0d4239f&quot;,&quot;shopId&quot;:&quot;11c8f677-9fe0-4fc8-b895-76e4b86e5dd9&quot;,&quot;userName&quot;:&quot;15658890633&quot;,&quot;password&quot;:&quot;32005beb6cc4314fe07aa6b0c9208f99&quot;}
{
	&quot;code&quot;:200,
	&quot;data&quot;:{
		&quot;employeeId&quot;:&quot;b43c796e-1b43-45e8-a2bb-d84dd01ad034&quot;,
		&quot;shops&quot;:null,
		&quot;token&quot;:&quot;ccefa4c6-4508-4dce-a453-170b35719942&quot;
	},
	&quot;message&quot;:&quot;&quot;,
	&quot;success&quot;:true
}
</system-out>
  </testcase>
  <testcase time="0.878" classname="main.testcase.RegisterAndlogin.login" name="login"/>
  <testcase time="0.928" classname="main.testcase.RegisterAndlogin.login" name="login"/>
  <testcase time="0.88" classname="main.testcase.RegisterAndlogin.login" name="login"/>
  <testcase time="0.17" classname="main.testcase.RegisterAndlogin.getAuthCode" name="getAuthCode_code"/>
  <testcase time="0.031" classname="main.testcase.RegisterAndlogin.getAuthCode" name="getAuthCode_code"/>
  <testcase time="0.029" classname="main.testcase.RegisterAndlogin.getAuthCode" name="getAuthCode_code"/>
  <testcase time="0.029" classname="main.testcase.RegisterAndlogin.getAuthCode" name="getAuthCode_code"/>
  <testcase time="0.027" classname="main.testcase.RegisterAndlogin.getAuthCode" name="getAuthCode_code"/>
  <testcase time="0.062" classname="main.testcase.RegisterAndlogin.getAuthCode" name="getAuthCode_code"/>
  <testcase time="0.029" classname="main.testcase.RegisterAndlogin.getAuthCode" name="getAuthCode_code"/>
  <testcase time="0.807" classname="main.testcase.RegisterAndlogin.register" name="register_result"/>
  <testcase time="0.832" classname="main.testcase.RegisterAndlogin.register" name="register_result">
    <failure message="expected [空] but found []" type="java.lang.AssertionError">java.lang.AssertionError: expected [空] but found []
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at main.testcase.RegisterAndlogin.register.register_result(register.java:87)
</failure>
    <system-out>{&quot;systemVersion&quot;:&quot;1&quot;,&quot;appVersion&quot;:&quot;1&quot;,&quot;clientIP&quot;:&quot;192.168.9.55&quot;,&quot;deviceNumber&quot;:&quot;a750a6d6-edf1-4704-9146-bc7eed651d65&quot;,&quot;sessionGuid&quot;:&quot;b43c796e-1b43-45e8-a2bb-d84dd01ad034&quot;,&quot;token&quot;:&quot;ccefa4c6-4508-4dce-a453-170b35719942&quot;,&quot;shopId&quot;:&quot;11c8f677-9fe0-4fc8-b895-76e4b86e5dd9&quot;,&quot;username&quot;:&quot;15658890659&quot;,&quot;password&quot;:&quot;32005beb6cc4314fe07aa6b0c9208f99&quot;,&quot;captcha&quot;:&quot;11111&quot;}
{
	&quot;code&quot;:0,
	&quot;data&quot;:{
		&quot;employeeId&quot;:&quot;c322318f-283b-4c94-a2b3-cd5711304b9d&quot;,
		&quot;shops&quot;:null,
		&quot;token&quot;:&quot;9f810412-0e84-4fda-ba79-8a214549cc5c&quot;
	},
	&quot;message&quot;:&quot;&quot;,
	&quot;success&quot;:true
}
</system-out>
  </testcase>
  <testcase time="0.817" classname="main.testcase.RegisterAndlogin.register" name="register_result"/>
  <testcase time="0.8" classname="main.testcase.RegisterAndlogin.register" name="register_result">
    <failure message="expected [601] but found [10902]" type="java.lang.AssertionError">java.lang.AssertionError: expected [601] but found [10902]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:389)
	at org.testng.Assert.assertEquals(Assert.java:399)
	at main.testcase.RegisterAndlogin.register.register_result(register.java:85)
</failure>
    <system-out>{&quot;systemVersion&quot;:&quot;1&quot;,&quot;appVersion&quot;:&quot;1&quot;,&quot;clientIP&quot;:&quot;192.168.9.55&quot;,&quot;deviceNumber&quot;:&quot;a750a6d6-edf1-4704-9146-bc7eed651d65&quot;,&quot;sessionGuid&quot;:&quot;b43c796e-1b43-45e8-a2bb-d84dd01ad034&quot;,&quot;token&quot;:&quot;ccefa4c6-4508-4dce-a453-170b35719942&quot;,&quot;shopId&quot;:&quot;11c8f677-9fe0-4fc8-b895-76e4b86e5dd9&quot;,&quot;username&quot;:&quot;15658890657&quot;,&quot;password&quot;:&quot;32005beb6cc4314fe07aa6b0c9208f99&quot;,&quot;captcha&quot;:&quot;空&quot;}
{
	&quot;code&quot;:10902,
	&quot;data&quot;:null,
	&quot;message&quot;:&quot;验证码错误&quot;,
	&quot;success&quot;:false
}
</system-out>
  </testcase>
  <testcase time="0.835" classname="main.testcase.RegisterAndlogin.register" name="register_result">
    <failure message="expected [601] but found [0]" type="java.lang.AssertionError">java.lang.AssertionError: expected [601] but found [0]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:389)
	at org.testng.Assert.assertEquals(Assert.java:399)
	at main.testcase.RegisterAndlogin.register.register_result(register.java:85)
</failure>
    <system-out>{&quot;systemVersion&quot;:&quot;1&quot;,&quot;appVersion&quot;:&quot;1&quot;,&quot;clientIP&quot;:&quot;192.168.9.55&quot;,&quot;deviceNumber&quot;:&quot;a750a6d6-edf1-4704-9146-bc7eed651d65&quot;,&quot;sessionGuid&quot;:&quot;b43c796e-1b43-45e8-a2bb-d84dd01ad034&quot;,&quot;token&quot;:&quot;ccefa4c6-4508-4dce-a453-170b35719942&quot;,&quot;shopId&quot;:&quot;11c8f677-9fe0-4fc8-b895-76e4b86e5dd9&quot;,&quot;username&quot;:&quot;15658890656&quot;,&quot;password&quot;:&quot;空&quot;,&quot;captcha&quot;:&quot;11111&quot;}
{
	&quot;code&quot;:0,
	&quot;data&quot;:{
		&quot;employeeId&quot;:&quot;89010765-48a8-4642-a702-e973a58d7947&quot;,
		&quot;shops&quot;:null,
		&quot;token&quot;:&quot;1fd7f64e-7d3c-41fd-a703-0fed1d832928&quot;
	},
	&quot;message&quot;:&quot;&quot;,
	&quot;success&quot;:true
}
</system-out>
  </testcase>
  <testcase time="0.817" classname="main.testcase.RegisterAndlogin.register" name="register_result">
    <failure message="expected [601] but found [0]" type="java.lang.AssertionError">java.lang.AssertionError: expected [601] but found [0]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:389)
	at org.testng.Assert.assertEquals(Assert.java:399)
	at main.testcase.RegisterAndlogin.register.register_result(register.java:85)
</failure>
    <system-out>{&quot;systemVersion&quot;:&quot;1&quot;,&quot;appVersion&quot;:&quot;1&quot;,&quot;clientIP&quot;:&quot;192.168.9.55&quot;,&quot;deviceNumber&quot;:&quot;a750a6d6-edf1-4704-9146-bc7eed651d65&quot;,&quot;sessionGuid&quot;:&quot;b43c796e-1b43-45e8-a2bb-d84dd01ad034&quot;,&quot;token&quot;:&quot;ccefa4c6-4508-4dce-a453-170b35719942&quot;,&quot;shopId&quot;:&quot;11c8f677-9fe0-4fc8-b895-76e4b86e5dd9&quot;,&quot;username&quot;:&quot;空&quot;,&quot;password&quot;:&quot;32005beb6cc4314fe07aa6b0c9208f99&quot;,&quot;captcha&quot;:&quot;11111&quot;}
{
	&quot;code&quot;:0,
	&quot;data&quot;:{
		&quot;employeeId&quot;:&quot;51e336c1-4966-4046-97e0-6c86b5dc00d3&quot;,
		&quot;shops&quot;:null,
		&quot;token&quot;:&quot;4deba855-070e-4db5-95d9-449c735baa01&quot;
	},
	&quot;message&quot;:&quot;&quot;,
	&quot;success&quot;:true
}
</system-out>
  </testcase>
  <testcase time="0.86" classname="main.testcase.RegisterAndlogin.completeRegistrationClerk" name="cp_result"/>
  <testcase time="0.852" classname="main.testcase.RegisterAndlogin.completeRegistrationClerk" name="cp_result"/>
  <testcase time="0.819" classname="main.testcase.RegisterAndlogin.completeRegistrationClerk" name="cp_result"/>
  <testcase time="0.814" classname="main.testcase.RegisterAndlogin.completeRegistrationClerk" name="cp_result"/>
  <testcase time="0.842" classname="main.testcase.RegisterAndlogin.completeRegistrationClerk" name="cp_result">
    <failure message="expected [601] but found [200]" type="java.lang.AssertionError">java.lang.AssertionError: expected [601] but found [200]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:389)
	at org.testng.Assert.assertEquals(Assert.java:399)
	at main.testcase.RegisterAndlogin.completeRegistrationClerk.cp_result(completeRegistrationClerk.java:74)
</failure>
    <system-out>{&quot;systemVersion&quot;:&quot;1&quot;,&quot;appVersion&quot;:&quot;1&quot;,&quot;clientIP&quot;:&quot;192.168.9.55&quot;,&quot;deviceNumber&quot;:&quot;a750a6d6-edf1-4704-9146-bc7eed651d65&quot;,&quot;sessionGuid&quot;:&quot;b43c796e-1b43-45e8-a2bb-d84dd01ad034&quot;,&quot;token&quot;:&quot;ccefa4c6-4508-4dce-a453-170b35719942&quot;,&quot;shopId&quot;:&quot;11c8f677-9fe0-4fc8-b895-76e4b86e5dd9&quot;,&quot;name&quot;:&quot;空&quot;,&quot;birthday&quot;:&quot;1990-12-13&quot;,&quot;sex&quot;:1}
{
	&quot;code&quot;:200,
	&quot;data&quot;:{
		&quot;shops&quot;:null
	},
	&quot;message&quot;:&quot;&quot;,
	&quot;success&quot;:true
}
</system-out>
  </testcase>
  <testcase time="0" classname="main.testcase.RegisterAndlogin.completeRegistrationClerk" name="cp_result">
    <failure message="For input string: &quot;空&quot;" type="java.lang.NumberFormatException">java.lang.NumberFormatException: For input string: &quot;空&quot;
	at java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
	at java.lang.Integer.parseInt(Integer.java:492)
	at java.lang.Integer.parseInt(Integer.java:527)
	at main.testcase.RegisterAndlogin.completeRegistrationClerk.cp_result(completeRegistrationClerk.java:58)
</failure>
  </testcase>
  <testcase time="0.826" classname="main.testcase.RegisterAndlogin.completeRegistrationClerk" name="cp_result"/>
  <testcase time="0.69" classname="main.testcase.RegisterAndlogin.completeRegistrationClerk" name="cp_result"/>
  <testcase time="0.79" classname="main.testcase.RegisterAndlogin.completeRegistrationClerk" name="cp_result"/>
  <testcase time="0.047" classname="main.testcase.RegisterAndlogin.managerCompleteRegistration" name="mcr_exresult">
    <failure message="expected [200] but found [601]" type="java.lang.AssertionError">java.lang.AssertionError: expected [200] but found [601]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:389)
	at org.testng.Assert.assertEquals(Assert.java:399)
	at main.testcase.RegisterAndlogin.managerCompleteRegistration.mcr_exresult(managerCompleteRegistration.java:79)
</failure>
    <system-out>{&quot;deviceType&quot;:1,&quot;systemVersion&quot;:1,&quot;appVersion&quot;:1,&quot;clientIP&quot;:&quot;192.168.9.55&quot;,&quot;deviceNumber&quot;:&quot;a750a6d6-edf1-4704-9146-bc7eed651d65&quot;,&quot;token&quot;:&quot;ccefa4c6-4508-4dce-a453-170b35719942&quot;,&quot;sessionGuid&quot;:&quot;b43c796e-1b43-45e8-a2bb-d84dd01ad034&quot;,&quot;shopName&quot;:&quot;测试店1&quot;,&quot;cityId&quot;:&quot;003fa057-a870-4a94-9ac2-d01afc79df37&quot;,&quot;address&quot;:&quot;浙江杭州&quot;}
{&quot;success&quot;:false,&quot;code&quot;:601,&quot;message&quot;:&quot;signImage can not be empty.&quot;}
</system-out>
  </testcase>
  <testcase time="0.042" classname="main.testcase.RegisterAndlogin.managerCompleteRegistration" name="mcr_exresult">
    <failure message="expected [10006] but found [601]" type="java.lang.AssertionError">java.lang.AssertionError: expected [10006] but found [601]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:389)
	at org.testng.Assert.assertEquals(Assert.java:399)
	at main.testcase.RegisterAndlogin.managerCompleteRegistration.mcr_exresult(managerCompleteRegistration.java:79)
</failure>
    <system-out>{&quot;deviceType&quot;:1,&quot;systemVersion&quot;:1,&quot;appVersion&quot;:1,&quot;clientIP&quot;:&quot;192.168.9.55&quot;,&quot;deviceNumber&quot;:&quot;a750a6d6-edf1-4704-9146-bc7eed651d65&quot;,&quot;token&quot;:&quot;ccefa4c6-4508-4dce-a453-170b35719942&quot;,&quot;sessionGuid&quot;:&quot;b43c796e-1b43-45e8-a2bb-d84dd01ad034&quot;,&quot;shopName&quot;:&quot;哦哦&quot;,&quot;cityId&quot;:&quot;003fa057-a870-4a94-9ac2-d01afc79df37&quot;,&quot;address&quot;:&quot;浙江杭州&quot;}
{&quot;success&quot;:false,&quot;code&quot;:601,&quot;message&quot;:&quot;signImage can not be empty.&quot;}
</system-out>
  </testcase>
  <testcase time="0.043" classname="main.testcase.RegisterAndlogin.managerCompleteRegistration" name="mcr_exresult">
    <failure message="expected [店铺名称太长] but found [signImage can not be empty.]" type="java.lang.AssertionError">java.lang.AssertionError: expected [店铺名称太长] but found [signImage can not be empty.]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at main.testcase.RegisterAndlogin.managerCompleteRegistration.mcr_exresult(managerCompleteRegistration.java:81)
</failure>
    <system-out>{&quot;deviceType&quot;:1,&quot;systemVersion&quot;:1,&quot;appVersion&quot;:1,&quot;clientIP&quot;:&quot;192.168.9.55&quot;,&quot;deviceNumber&quot;:&quot;a750a6d6-edf1-4704-9146-bc7eed651d65&quot;,&quot;token&quot;:&quot;ccefa4c6-4508-4dce-a453-170b35719942&quot;,&quot;sessionGuid&quot;:&quot;b43c796e-1b43-45e8-a2bb-d84dd01ad034&quot;,&quot;shopName&quot;:&quot;名字超长名字超长名字超长名字超长名字超长名字超长名字超长名字超长2&quot;,&quot;cityId&quot;:&quot;003fa057-a870-4a94-9ac2-d01afc79df37&quot;,&quot;address&quot;:&quot;浙江杭州&quot;}
{&quot;success&quot;:false,&quot;code&quot;:601,&quot;message&quot;:&quot;signImage can not be empty.&quot;}
</system-out>
  </testcase>
  <testcase time="0.047" classname="main.testcase.RegisterAndlogin.managerCompleteRegistration" name="mcr_exresult">
    <failure message="expected [地址参数长度错误] but found [signImage can not be empty.]" type="java.lang.AssertionError">java.lang.AssertionError: expected [地址参数长度错误] but found [signImage can not be empty.]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at main.testcase.RegisterAndlogin.managerCompleteRegistration.mcr_exresult(managerCompleteRegistration.java:81)
</failure>
    <system-out>{&quot;deviceType&quot;:1,&quot;systemVersion&quot;:1,&quot;appVersion&quot;:1,&quot;clientIP&quot;:&quot;192.168.9.55&quot;,&quot;deviceNumber&quot;:&quot;a750a6d6-edf1-4704-9146-bc7eed651d65&quot;,&quot;token&quot;:&quot;ccefa4c6-4508-4dce-a453-170b35719942&quot;,&quot;sessionGuid&quot;:&quot;b43c796e-1b43-45e8-a2bb-d84dd01ad034&quot;,&quot;shopName&quot;:&quot;测试店1&quot;,&quot;cityId&quot;:&quot;003fa057-a870-4a94-9ac2-d01afc79df37&quot;,&quot;address&quot;:&quot;地址超长地址超长地址超长地址超长地址超长地址超长地址超长地址超长地址超长地址超长地址超长超长地址超长地址超长地址超长地址超长地址超长超长地址超长地址超长地址超长地址超长地址超长&quot;}
{&quot;success&quot;:false,&quot;code&quot;:601,&quot;message&quot;:&quot;signImage can not be empty.&quot;}
</system-out>
  </testcase>
  <testcase time="0.046" classname="main.testcase.RegisterAndlogin.managerCompleteRegistration" name="mcr_exresult">
    <failure message="expected [200] but found [601]" type="java.lang.AssertionError">java.lang.AssertionError: expected [200] but found [601]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:389)
	at org.testng.Assert.assertEquals(Assert.java:399)
	at main.testcase.RegisterAndlogin.managerCompleteRegistration.mcr_exresult(managerCompleteRegistration.java:79)
</failure>
    <system-out>{&quot;deviceType&quot;:1,&quot;systemVersion&quot;:1,&quot;appVersion&quot;:1,&quot;clientIP&quot;:&quot;192.168.9.55&quot;,&quot;deviceNumber&quot;:&quot;a750a6d6-edf1-4704-9146-bc7eed651d65&quot;,&quot;token&quot;:&quot;ccefa4c6-4508-4dce-a453-170b35719942&quot;,&quot;sessionGuid&quot;:&quot;b43c796e-1b43-45e8-a2bb-d84dd01ad034&quot;,&quot;shopName&quot;:&quot;测试店5&quot;,&quot;cityId&quot;:&quot;003fa057-1111-1111-1111-d01afc79df37&quot;,&quot;address&quot;:&quot;浙江杭州&quot;}
{&quot;success&quot;:false,&quot;code&quot;:601,&quot;message&quot;:&quot;signImage can not be empty.&quot;}
</system-out>
  </testcase>
  <testcase time="0.046" classname="main.testcase.RegisterAndlogin.managerCompleteRegistration" name="mcr_exresult">
    <failure message="expected [店铺名称不能为空] but found [signImage can not be empty.]" type="java.lang.AssertionError">java.lang.AssertionError: expected [店铺名称不能为空] but found [signImage can not be empty.]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at main.testcase.RegisterAndlogin.managerCompleteRegistration.mcr_exresult(managerCompleteRegistration.java:81)
</failure>
    <system-out>{&quot;deviceType&quot;:1,&quot;systemVersion&quot;:1,&quot;appVersion&quot;:1,&quot;clientIP&quot;:&quot;192.168.9.55&quot;,&quot;deviceNumber&quot;:&quot;a750a6d6-edf1-4704-9146-bc7eed651d65&quot;,&quot;token&quot;:&quot;ccefa4c6-4508-4dce-a453-170b35719942&quot;,&quot;sessionGuid&quot;:&quot;b43c796e-1b43-45e8-a2bb-d84dd01ad034&quot;,&quot;shopName&quot;:&quot;空&quot;,&quot;cityId&quot;:&quot;003fa057-a870-4a94-9ac2-d01afc79df37&quot;,&quot;address&quot;:&quot;浙江杭州&quot;}
{&quot;success&quot;:false,&quot;code&quot;:601,&quot;message&quot;:&quot;signImage can not be empty.&quot;}
</system-out>
  </testcase>
  <testcase time="0.049" classname="main.testcase.RegisterAndlogin.managerCompleteRegistration" name="mcr_exresult">
    <failure message="expected [城市参数长度错误] but found [signImage can not be empty.]" type="java.lang.AssertionError">java.lang.AssertionError: expected [城市参数长度错误] but found [signImage can not be empty.]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at main.testcase.RegisterAndlogin.managerCompleteRegistration.mcr_exresult(managerCompleteRegistration.java:81)
</failure>
    <system-out>{&quot;deviceType&quot;:1,&quot;systemVersion&quot;:1,&quot;appVersion&quot;:1,&quot;clientIP&quot;:&quot;192.168.9.55&quot;,&quot;deviceNumber&quot;:&quot;a750a6d6-edf1-4704-9146-bc7eed651d65&quot;,&quot;token&quot;:&quot;ccefa4c6-4508-4dce-a453-170b35719942&quot;,&quot;sessionGuid&quot;:&quot;b43c796e-1b43-45e8-a2bb-d84dd01ad034&quot;,&quot;shopName&quot;:&quot;测试店6&quot;,&quot;cityId&quot;:&quot;空&quot;,&quot;address&quot;:&quot;浙江杭州&quot;}
{&quot;success&quot;:false,&quot;code&quot;:601,&quot;message&quot;:&quot;signImage can not be empty.&quot;}
</system-out>
  </testcase>
  <testcase time="0.047" classname="main.testcase.RegisterAndlogin.managerCompleteRegistration" name="mcr_exresult">
    <failure message="expected [地址参数长度错误] but found [signImage can not be empty.]" type="java.lang.AssertionError">java.lang.AssertionError: expected [地址参数长度错误] but found [signImage can not be empty.]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at main.testcase.RegisterAndlogin.managerCompleteRegistration.mcr_exresult(managerCompleteRegistration.java:81)
</failure>
    <system-out>{&quot;deviceType&quot;:1,&quot;systemVersion&quot;:1,&quot;appVersion&quot;:1,&quot;clientIP&quot;:&quot;192.168.9.55&quot;,&quot;deviceNumber&quot;:&quot;a750a6d6-edf1-4704-9146-bc7eed651d65&quot;,&quot;token&quot;:&quot;ccefa4c6-4508-4dce-a453-170b35719942&quot;,&quot;sessionGuid&quot;:&quot;b43c796e-1b43-45e8-a2bb-d84dd01ad034&quot;,&quot;shopName&quot;:&quot;测试店7&quot;,&quot;cityId&quot;:&quot;003fa057-a870-4a94-9ac2-d01afc79df37&quot;,&quot;address&quot;:&quot;空&quot;}
{&quot;success&quot;:false,&quot;code&quot;:601,&quot;message&quot;:&quot;signImage can not be empty.&quot;}
</system-out>
  </testcase>
  <testcase time="0.039" classname="main.testcase.RegisterAndlogin.checkShopName" name="checkHsopName"/>
  <testcase time="0.101" classname="main.testcase.RegisterAndlogin.checkShopName" name="checkHsopName"/>
  <testcase time="0.078" classname="main.testcase.RegisterAndlogin.checkShopName" name="checkHsopName">
    <failure message="expected [601] but found [200]" type="java.lang.AssertionError">java.lang.AssertionError: expected [601] but found [200]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:389)
	at org.testng.Assert.assertEquals(Assert.java:399)
	at main.testcase.RegisterAndlogin.checkShopName.checkHsopName(checkShopName.java:62)
</failure>
    <system-out>http://192.168.1.16:80/api/shop/name/checking?shopName=空&amp;sessionGuid=b43c796e-1b43-45e8-a2bb-d84dd01ad034&amp;shopId=11c8f677-9fe0-4fc8-b895-76e4b86e5dd9&amp;systemVersion=1&amp;clientIP=192.168.9.55&amp;deviceNumber=c812c5cf-e48d-4dc1-9ee3-e3b4e99fa2b1&amp;token=ccefa4c6-4508-4dce-a453-170b35719942&amp;appVersion=1
{
	&quot;code&quot;:200,
	&quot;data&quot;:null,
	&quot;message&quot;:&quot;&quot;,
	&quot;success&quot;:true
}
</system-out>
  </testcase>
  <testcase time="0.729" classname="main.testcase.RegisterAndlogin.retrievePassword" name="rp_result">
    <failure message="expected [601] but found [200]" type="java.lang.AssertionError">java.lang.AssertionError: expected [601] but found [200]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:389)
	at org.testng.Assert.assertEquals(Assert.java:399)
	at main.testcase.RegisterAndlogin.retrievePassword.rp_result(retrievePassword.java:71)
</failure>
    <system-out>{&quot;systemVersion&quot;:&quot;1&quot;,&quot;appVersion&quot;:&quot;1&quot;,&quot;clientIP&quot;:&quot;192.168.9.55&quot;,&quot;deviceNumber&quot;:&quot;a750a6d6-edf1-4704-9146-bc7eed651d65&quot;,&quot;sessionGuid&quot;:&quot;b43c796e-1b43-45e8-a2bb-d84dd01ad034&quot;,&quot;token&quot;:&quot;ccefa4c6-4508-4dce-a453-170b35719942&quot;,&quot;shopId&quot;:&quot;11c8f677-9fe0-4fc8-b895-76e4b86e5dd9&quot;,&quot;username&quot;:&quot;15658890633&quot;,&quot;captcha&quot;:&quot;11111&quot;,&quot;password&quot;:&quot;空&quot;}
{
	&quot;code&quot;:200,
	&quot;data&quot;:null,
	&quot;message&quot;:&quot;密码重置成功&quot;,
	&quot;success&quot;:true
}
</system-out>
  </testcase>
  <testcase time="0.749" classname="main.testcase.RegisterAndlogin.retrievePassword" name="rp_result"/>
  <testcase time="0.808" classname="main.testcase.RegisterAndlogin.retrievePassword" name="rp_result">
    <failure message="expected [601] but found [200]" type="java.lang.AssertionError">java.lang.AssertionError: expected [601] but found [200]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:389)
	at org.testng.Assert.assertEquals(Assert.java:399)
	at main.testcase.RegisterAndlogin.retrievePassword.rp_result(retrievePassword.java:71)
</failure>
    <system-out>{&quot;systemVersion&quot;:&quot;1&quot;,&quot;appVersion&quot;:&quot;1&quot;,&quot;clientIP&quot;:&quot;192.168.9.55&quot;,&quot;deviceNumber&quot;:&quot;a750a6d6-edf1-4704-9146-bc7eed651d65&quot;,&quot;sessionGuid&quot;:&quot;b43c796e-1b43-45e8-a2bb-d84dd01ad034&quot;,&quot;token&quot;:&quot;ccefa4c6-4508-4dce-a453-170b35719942&quot;,&quot;shopId&quot;:&quot;11c8f677-9fe0-4fc8-b895-76e4b86e5dd9&quot;,&quot;username&quot;:&quot;空&quot;,&quot;captcha&quot;:&quot;11111&quot;,&quot;password&quot;:&quot;32005beb6cc4314fe07aa6b0c9208f99&quot;}
{
	&quot;code&quot;:200,
	&quot;data&quot;:null,
	&quot;message&quot;:&quot;密码重置成功&quot;,
	&quot;success&quot;:true
}
</system-out>
  </testcase>
  <testcase time="0.809" classname="main.testcase.RegisterAndlogin.retrievePassword" name="rp_result">
    <failure message="expected [601] but found [10902]" type="java.lang.AssertionError">java.lang.AssertionError: expected [601] but found [10902]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:389)
	at org.testng.Assert.assertEquals(Assert.java:399)
	at main.testcase.RegisterAndlogin.retrievePassword.rp_result(retrievePassword.java:71)
</failure>
    <system-out>{&quot;systemVersion&quot;:&quot;1&quot;,&quot;appVersion&quot;:&quot;1&quot;,&quot;clientIP&quot;:&quot;192.168.9.55&quot;,&quot;deviceNumber&quot;:&quot;a750a6d6-edf1-4704-9146-bc7eed651d65&quot;,&quot;sessionGuid&quot;:&quot;b43c796e-1b43-45e8-a2bb-d84dd01ad034&quot;,&quot;token&quot;:&quot;ccefa4c6-4508-4dce-a453-170b35719942&quot;,&quot;shopId&quot;:&quot;11c8f677-9fe0-4fc8-b895-76e4b86e5dd9&quot;,&quot;username&quot;:&quot;15658890633&quot;,&quot;captcha&quot;:&quot;空&quot;,&quot;password&quot;:&quot;32005beb6cc4314fe07aa6b0c9208f99&quot;}
{
	&quot;code&quot;:10902,
	&quot;data&quot;:null,
	&quot;message&quot;:&quot;验证码错误&quot;,
	&quot;success&quot;:false
}
</system-out>
  </testcase>
  <testcase time="0.815" classname="main.testcase.RegisterAndlogin.retrievePassword" name="rp_result"/>
  <testcase time="0.712" classname="main.testcase.RegisterAndlogin.retrievePassword" name="rp_result"/>
  <testcase time="0.789" classname="main.testcase.RegisterAndlogin.retrievePassword" name="rp_result"/>
  <testcase time="0.81" classname="main.testcase.RegisterAndlogin.retrievePassword" name="rp_result"/>
  <testcase time="0.797" classname="main.testcase.RegisterAndlogin.retrievePassword" name="rp_result"/>
  <testcase time="0.062" classname="main.testcase.GatheringList.gatheringList" name="Gl"/>
  <testcase time="0.028" classname="main.testcase.GatheringList.gatheringList" name="Gl"/>
  <testcase time="0.028" classname="main.testcase.GatheringList.gatheringList" name="Gl"/>
  <testcase time="0.027" classname="main.testcase.GatheringList.gatheringList" name="Gl"/>
  <testcase time="0.843" classname="main.testcase.GatheringList.collectionSchedule" name="collectionSchedule"/>
  <testcase time="0.687" classname="main.testcase.GatheringList.collectionSchedule" name="collectionSchedule"/>
  <testcase time="0.715" classname="main.testcase.GatheringList.collectionSchedule" name="collectionSchedule"/>
  <testcase time="0.678" classname="main.testcase.GatheringList.collectionSchedule" name="collectionSchedule"/>
  <testcase time="0.799" classname="main.testcase.GatheringList.collectionSchedule" name="collectionSchedule"/>
  <testcase time="0.802" classname="main.testcase.GatheringList.collectionSchedule" name="collectionSchedule"/>
  <testcase time="0.045" classname="main.testcase.Setting.personalDataToSee" name="pdte"/>
  <testcase time="0.068" classname="main.testcase.Setting.changePassword" name="cp_code">
    <failure message="expected [空] but found []" type="java.lang.AssertionError">java.lang.AssertionError: expected [空] but found []
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at main.testcase.Setting.changePassword.cp_code(changePassword.java:86)
</failure>
    <system-out>{&quot;deviceType&quot;:1,&quot;systemVersion&quot;:1,&quot;appVersion&quot;:1,&quot;clientIP&quot;:&quot;192.168.9.55&quot;,&quot;deviceNumber&quot;:&quot;a750a6d6-edf1-4704-9146-bc7eed651d65&quot;,&quot;token&quot;:&quot;ccefa4c6-4508-4dce-a453-170b35719942&quot;,&quot;sessionGuid&quot;:&quot;b43c796e-1b43-45e8-a2bb-d84dd01ad034&quot;,&quot;oldPassword&quot;:&quot;32005beb6cc4314fe07aa6b0c9208f99&quot;,&quot;newPasswordassword&quot;:&quot;32005beb6cc4314fe07aa6b0c9208f99&quot;}
{
	&quot;code&quot;:200,
	&quot;data&quot;:null,
	&quot;message&quot;:&quot;&quot;,
	&quot;success&quot;:true
}
</system-out>
  </testcase>
  <testcase time="0.052" classname="main.testcase.Setting.personalCenter" name="personCentre_result"/>
  <testcase time="0.184" classname="main.testcase.Setting.personalCenter" name="personCentre_result"/>
  <testcase time="0.185" classname="main.testcase.Setting.personalCenter" name="personCentre_result"/>
  <testcase time="0.183" classname="main.testcase.Setting.personalCenter" name="personCentre_result"/>
  <testcase time="0.074" classname="main.testcase.Wallet.accountBalance" name="ab"/>
  <testcase time="0.024" classname="main.testcase.Wallet.accountBalance" name="ab">
    <failure type="java.lang.NullPointerException">java.lang.NullPointerException
	at main.testcase.Wallet.accountBalance.ab(accountBalance.java:69)
</failure>
    <system-out>http://192.168.1.16:80/api/ucenter/balance/shop?shopId=11c8f677-1111-1111-b895-76e4b86e5dd1&amp;systemVersion=1&amp;clientIP=192.168.9.55&amp;deviceNumber=c812c5cf-e48d-4dc1-9ee3-e3b4e99fa2b1&amp;sessionGuid=b43c796e-1b43-45e8-a2bb-d84dd01ad034&amp;token=ccefa4c6-4508-4dce-a453-170b35719942&amp;appVersion=1
{&quot;success&quot;:false,&quot;code&quot;:403,&quot;message&quot;:&quot;Sorry, there is no employee shop relations in cache!&quot;}
</system-out>
  </testcase>
  <testcase time="0.025" classname="main.testcase.Wallet.accountBalance" name="ab">
    <failure type="java.lang.NullPointerException">java.lang.NullPointerException
	at main.testcase.Wallet.accountBalance.ab(accountBalance.java:69)
</failure>
    <system-out>http://192.168.1.16:80/api/ucenter/balance/shop?shopId=11c8f677-9fe0-4fc8-b895-222222222222222222&amp;systemVersion=1&amp;clientIP=192.168.9.55&amp;deviceNumber=c812c5cf-e48d-4dc1-9ee3-e3b4e99fa2b1&amp;sessionGuid=b43c796e-1b43-45e8-a2bb-d84dd01ad034&amp;token=ccefa4c6-4508-4dce-a453-170b35719942&amp;appVersion=1
{&quot;success&quot;:false,&quot;code&quot;:403,&quot;message&quot;:&quot;Sorry, there is no employee shop relations in cache!&quot;}
</system-out>
  </testcase>
  <testcase time="0.025" classname="main.testcase.Wallet.accountBalance" name="ab">
    <failure type="java.lang.NullPointerException">java.lang.NullPointerException
	at main.testcase.Wallet.accountBalance.ab(accountBalance.java:69)
</failure>
    <system-out>http://192.168.1.16:80/api/ucenter/balance/shop?shopId=11c8f677-9fe0-4fc8-b895-7&amp;systemVersion=1&amp;clientIP=192.168.9.55&amp;deviceNumber=c812c5cf-e48d-4dc1-9ee3-e3b4e99fa2b1&amp;sessionGuid=b43c796e-1b43-45e8-a2bb-d84dd01ad034&amp;token=ccefa4c6-4508-4dce-a453-170b35719942&amp;appVersion=1
{&quot;success&quot;:false,&quot;code&quot;:403,&quot;message&quot;:&quot;Sorry, there is no employee shop relations in cache!&quot;}
</system-out>
  </testcase>
  <testcase time="0.044" classname="main.testcase.Wallet.withdrawalRecordList" name="wrl_code"/>
  <testcase time="1.046" classname="main.testcase.Wallet.BeforeTheWithdrawalInformationQuery" name="btwiq"/>
  <testcase time="1.025" classname="main.testcase.Wallet.BeforeTheWithdrawalInformationQuery" name="btwiq"/>
  <testcase time="1.027" classname="main.testcase.Wallet.BeforeTheWithdrawalInformationQuery" name="btwiq"/>
  <testcase time="1.042" classname="main.testcase.Wallet.BeforeTheWithdrawalInformationQuery" name="btwiq"/>
  <testcase time="0.777" classname="main.testcase.Wallet.Withdraw" name="withdraw"/>
  <testcase time="0.72" classname="main.testcase.Wallet.Withdraw" name="withdraw"/>
  <testcase time="0.666" classname="main.testcase.Wallet.Withdraw" name="withdraw"/>
  <testcase time="0.724" classname="main.testcase.Wallet.Withdraw" name="withdraw"/>
  <testcase time="0.805" classname="main.testcase.Wallet.Withdraw" name="withdraw"/>
  <testcase time="0.805" classname="main.testcase.Wallet.Withdraw" name="withdraw">
    <failure message="expected [601] but found [10509]" type="java.lang.AssertionError">java.lang.AssertionError: expected [601] but found [10509]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:389)
	at org.testng.Assert.assertEquals(Assert.java:399)
	at main.testcase.Wallet.Withdraw.withdraw(Withdraw.java:89)
</failure>
    <system-out>{&quot;systemVersion&quot;:&quot;1&quot;,&quot;appVersion&quot;:&quot;1&quot;,&quot;clientIP&quot;:&quot;192.168.9.55&quot;,&quot;deviceNumber&quot;:&quot;a750a6d6-edf1-4704-9146-bc7eed651d65&quot;,&quot;sessionGuid&quot;:&quot;b43c796e-1b43-45e8-a2bb-d84dd01ad034&quot;,&quot;token&quot;:&quot;ccefa4c6-4508-4dce-a453-170b35719942&quot;,&quot;shopId&quot;:&quot;11c8f677-9fe0-4fc8-b895-76e4b86e5dd9&quot;,&quot;account&quot;:&quot;空&quot;,&quot;bankNo&quot;:&quot;123123456632123000&quot;,&quot;bankName&quot;:&quot;中国工商银行&quot;,&quot;branchName&quot;:&quot;杭州滨江支行&quot;,&quot;amount&quot;:1.52,&quot;password&quot;:&quot;32005beb6cc4314fe07aa6b0c9208f99&quot;}
{
	&quot;code&quot;:10509,
	&quot;data&quot;:null,
	&quot;message&quot;:&quot;用户未认证，无法进行提现,请认证后再进行提现！&quot;,
	&quot;success&quot;:false
}
</system-out>
  </testcase>
  <testcase time="1.426" classname="main.testcase.Wallet.Withdraw" name="withdraw"/>
  <testcase time="0.802" classname="main.testcase.Wallet.Withdraw" name="withdraw">
    <failure message="expected [601] but found [10509]" type="java.lang.AssertionError">java.lang.AssertionError: expected [601] but found [10509]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:389)
	at org.testng.Assert.assertEquals(Assert.java:399)
	at main.testcase.Wallet.Withdraw.withdraw(Withdraw.java:89)
</failure>
    <system-out>{&quot;systemVersion&quot;:&quot;1&quot;,&quot;appVersion&quot;:&quot;1&quot;,&quot;clientIP&quot;:&quot;192.168.9.55&quot;,&quot;deviceNumber&quot;:&quot;a750a6d6-edf1-4704-9146-bc7eed651d65&quot;,&quot;sessionGuid&quot;:&quot;b43c796e-1b43-45e8-a2bb-d84dd01ad034&quot;,&quot;token&quot;:&quot;ccefa4c6-4508-4dce-a453-170b35719942&quot;,&quot;shopId&quot;:&quot;11c8f677-9fe0-4fc8-b895-76e4b86e5dd9&quot;,&quot;account&quot;:&quot;汤涛&quot;,&quot;bankNo&quot;:&quot;123123456632123000&quot;,&quot;bankName&quot;:&quot;空&quot;,&quot;branchName&quot;:&quot;杭州滨江支行&quot;,&quot;amount&quot;:1.52,&quot;password&quot;:&quot;32005beb6cc4314fe07aa6b0c9208f99&quot;}
{
	&quot;code&quot;:10509,
	&quot;data&quot;:null,
	&quot;message&quot;:&quot;用户未认证，无法进行提现,请认证后再进行提现！&quot;,
	&quot;success&quot;:false
}
</system-out>
  </testcase>
  <testcase time="0.945" classname="main.testcase.Wallet.Withdraw" name="withdraw">
    <failure message="expected [601] but found [10509]" type="java.lang.AssertionError">java.lang.AssertionError: expected [601] but found [10509]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:389)
	at org.testng.Assert.assertEquals(Assert.java:399)
	at main.testcase.Wallet.Withdraw.withdraw(Withdraw.java:89)
</failure>
    <system-out>{&quot;systemVersion&quot;:&quot;1&quot;,&quot;appVersion&quot;:&quot;1&quot;,&quot;clientIP&quot;:&quot;192.168.9.55&quot;,&quot;deviceNumber&quot;:&quot;a750a6d6-edf1-4704-9146-bc7eed651d65&quot;,&quot;sessionGuid&quot;:&quot;b43c796e-1b43-45e8-a2bb-d84dd01ad034&quot;,&quot;token&quot;:&quot;ccefa4c6-4508-4dce-a453-170b35719942&quot;,&quot;shopId&quot;:&quot;11c8f677-9fe0-4fc8-b895-76e4b86e5dd9&quot;,&quot;account&quot;:&quot;汤涛&quot;,&quot;bankNo&quot;:&quot;123123456632123000&quot;,&quot;bankName&quot;:&quot;中国工商银行&quot;,&quot;branchName&quot;:&quot;空&quot;,&quot;amount&quot;:1.52,&quot;password&quot;:&quot;32005beb6cc4314fe07aa6b0c9208f99&quot;}
{
	&quot;code&quot;:10509,
	&quot;data&quot;:null,
	&quot;message&quot;:&quot;用户未认证，无法进行提现,请认证后再进行提现！&quot;,
	&quot;success&quot;:false
}
</system-out>
  </testcase>
  <testcase time="0.856" classname="main.testcase.Wallet.Withdraw" name="withdraw"/>
  <testcase time="0.745" classname="main.testcase.Wallet.Withdraw" name="withdraw"/>
  <testcase time="0.806" classname="main.testcase.Wallet.Withdraw" name="withdraw"/>
  <testcase time="0.675" classname="main.testcase.Wallet.Withdraw" name="withdraw"/>
  <testcase time="0.716" classname="main.testcase.Wallet.Withdraw" name="withdraw"/>
  <testcase time="1.017" classname="main.testcase.Wallet.Withdraw" name="withdraw"/>
  <testcase time="0.804" classname="main.testcase.Wallet.Withdraw" name="withdraw"/>
  <testcase time="0.823" classname="main.testcase.Wallet.Withdraw" name="withdraw"/>
  <testcase time="0.864" classname="main.testcase.Wallet.Withdraw" name="withdraw">
    <failure message="expected [601] but found [10509]" type="java.lang.AssertionError">java.lang.AssertionError: expected [601] but found [10509]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:389)
	at org.testng.Assert.assertEquals(Assert.java:399)
	at main.testcase.Wallet.Withdraw.withdraw(Withdraw.java:89)
</failure>
    <system-out>{&quot;systemVersion&quot;:&quot;1&quot;,&quot;appVersion&quot;:&quot;1&quot;,&quot;clientIP&quot;:&quot;192.168.9.55&quot;,&quot;deviceNumber&quot;:&quot;a750a6d6-edf1-4704-9146-bc7eed651d65&quot;,&quot;sessionGuid&quot;:&quot;b43c796e-1b43-45e8-a2bb-d84dd01ad034&quot;,&quot;token&quot;:&quot;ccefa4c6-4508-4dce-a453-170b35719942&quot;,&quot;shopId&quot;:&quot;11c8f677-9fe0-4fc8-b895-76e4b86e5dd9&quot;,&quot;account&quot;:&quot;汤涛&quot;,&quot;bankNo&quot;:&quot;123123456632123000&quot;,&quot;bankName&quot;:&quot;中国工商银行&quot;,&quot;branchName&quot;:&quot;杭州滨江支行杭州滨江支行杭州滨江支行杭州滨江支行杭州滨江支行杭州滨江支行杭州滨江支行杭州滨江支行杭州滨江支行&quot;,&quot;amount&quot;:1.52,&quot;password&quot;:&quot;32005beb6cc4314fe07aa6b0c9208f99&quot;}
{
	&quot;code&quot;:10509,
	&quot;data&quot;:null,
	&quot;message&quot;:&quot;用户未认证，无法进行提现,请认证后再进行提现！&quot;,
	&quot;success&quot;:false
}
</system-out>
  </testcase>
  <testcase time="0.719" classname="main.testcase.Wallet.withdraw_fail" name="w_al"/>
  <testcase time="0.755" classname="main.testcase.Wallet.withdraw_fail" name="w_al"/>
  <testcase time="0.785" classname="main.testcase.Wallet.withdraw_fail" name="w_al"/>
  <testcase time="0.881" classname="main.testcase.Wallet.withdraw_fail" name="w_al"/>
  <testcase time="0.789" classname="main.testcase.Wallet.withdraw_fail" name="w_al"/>
  <testcase time="0.705" classname="main.testcase.Wallet.withdraw_fail" name="w_al">
    <failure message="expected [601] but found [10509]" type="java.lang.AssertionError">java.lang.AssertionError: expected [601] but found [10509]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:389)
	at org.testng.Assert.assertEquals(Assert.java:399)
	at main.testcase.Wallet.withdraw_fail.w_al(withdraw_fail.java:85)
</failure>
    <system-out>{&quot;systemVersion&quot;:&quot;1&quot;,&quot;appVersion&quot;:&quot;1&quot;,&quot;clientIP&quot;:&quot;192.168.9.55&quot;,&quot;deviceNumber&quot;:&quot;a750a6d6-edf1-4704-9146-bc7eed651d65&quot;,&quot;sessionGuid&quot;:&quot;b43c796e-1b43-45e8-a2bb-d84dd01ad034&quot;,&quot;token&quot;:&quot;ccefa4c6-4508-4dce-a453-170b35719942&quot;,&quot;shopId&quot;:&quot;11c8f677-9fe0-4fc8-b895-76e4b86e5dd9&quot;,&quot;account&quot;:&quot;空&quot;,&quot;bankNo&quot;:&quot;123123456632123000&quot;,&quot;bankName&quot;:&quot;中国工商银行&quot;,&quot;branchName&quot;:&quot;杭州滨江支行&quot;,&quot;amount&quot;:1.52,&quot;password&quot;:&quot;32005beb6cc4314fe07aa6b0c9208f99&quot;}
{
	&quot;code&quot;:10509,
	&quot;data&quot;:null,
	&quot;message&quot;:&quot;用户未认证，无法进行提现,请认证后再进行提现！&quot;,
	&quot;success&quot;:false
}
</system-out>
  </testcase>
  <testcase time="0.748" classname="main.testcase.Wallet.withdraw_fail" name="w_al"/>
  <testcase time="0.854" classname="main.testcase.Wallet.withdraw_fail" name="w_al">
    <failure message="expected [601] but found [10509]" type="java.lang.AssertionError">java.lang.AssertionError: expected [601] but found [10509]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:389)
	at org.testng.Assert.assertEquals(Assert.java:399)
	at main.testcase.Wallet.withdraw_fail.w_al(withdraw_fail.java:85)
</failure>
    <system-out>{&quot;systemVersion&quot;:&quot;1&quot;,&quot;appVersion&quot;:&quot;1&quot;,&quot;clientIP&quot;:&quot;192.168.9.55&quot;,&quot;deviceNumber&quot;:&quot;a750a6d6-edf1-4704-9146-bc7eed651d65&quot;,&quot;sessionGuid&quot;:&quot;b43c796e-1b43-45e8-a2bb-d84dd01ad034&quot;,&quot;token&quot;:&quot;ccefa4c6-4508-4dce-a453-170b35719942&quot;,&quot;shopId&quot;:&quot;11c8f677-9fe0-4fc8-b895-76e4b86e5dd9&quot;,&quot;account&quot;:&quot;汤涛&quot;,&quot;bankNo&quot;:&quot;123123456632123000&quot;,&quot;bankName&quot;:&quot;空&quot;,&quot;branchName&quot;:&quot;杭州滨江支行&quot;,&quot;amount&quot;:1.52,&quot;password&quot;:&quot;32005beb6cc4314fe07aa6b0c9208f99&quot;}
{
	&quot;code&quot;:10509,
	&quot;data&quot;:null,
	&quot;message&quot;:&quot;用户未认证，无法进行提现,请认证后再进行提现！&quot;,
	&quot;success&quot;:false
}
</system-out>
  </testcase>
  <testcase time="0.795" classname="main.testcase.Wallet.withdraw_fail" name="w_al">
    <failure message="expected [601] but found [10509]" type="java.lang.AssertionError">java.lang.AssertionError: expected [601] but found [10509]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:389)
	at org.testng.Assert.assertEquals(Assert.java:399)
	at main.testcase.Wallet.withdraw_fail.w_al(withdraw_fail.java:85)
</failure>
    <system-out>{&quot;systemVersion&quot;:&quot;1&quot;,&quot;appVersion&quot;:&quot;1&quot;,&quot;clientIP&quot;:&quot;192.168.9.55&quot;,&quot;deviceNumber&quot;:&quot;a750a6d6-edf1-4704-9146-bc7eed651d65&quot;,&quot;sessionGuid&quot;:&quot;b43c796e-1b43-45e8-a2bb-d84dd01ad034&quot;,&quot;token&quot;:&quot;ccefa4c6-4508-4dce-a453-170b35719942&quot;,&quot;shopId&quot;:&quot;11c8f677-9fe0-4fc8-b895-76e4b86e5dd9&quot;,&quot;account&quot;:&quot;汤涛&quot;,&quot;bankNo&quot;:&quot;123123456632123000&quot;,&quot;bankName&quot;:&quot;中国工商银行&quot;,&quot;branchName&quot;:&quot;空&quot;,&quot;amount&quot;:1.52,&quot;password&quot;:&quot;32005beb6cc4314fe07aa6b0c9208f99&quot;}
{
	&quot;code&quot;:10509,
	&quot;data&quot;:null,
	&quot;message&quot;:&quot;用户未认证，无法进行提现,请认证后再进行提现！&quot;,
	&quot;success&quot;:false
}
</system-out>
  </testcase>
  <testcase time="0.816" classname="main.testcase.Wallet.withdraw_fail" name="w_al"/>
  <testcase time="0.798" classname="main.testcase.Wallet.withdraw_fail" name="w_al"/>
  <testcase time="0.675" classname="main.testcase.Wallet.withdraw_fail" name="w_al"/>
  <testcase time="0.733" classname="main.testcase.Wallet.withdraw_fail" name="w_al"/>
  <testcase time="0.823" classname="main.testcase.Wallet.withdraw_fail" name="w_al"/>
  <testcase time="0.827" classname="main.testcase.Wallet.withdraw_fail" name="w_al"/>
  <testcase time="0.811" classname="main.testcase.Wallet.withdraw_fail" name="w_al"/>
  <testcase time="0.781" classname="main.testcase.Wallet.withdraw_fail" name="w_al"/>
  <testcase time="0.663" classname="main.testcase.Wallet.withdraw_fail" name="w_al">
    <failure message="expected [601] but found [10509]" type="java.lang.AssertionError">java.lang.AssertionError: expected [601] but found [10509]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:389)
	at org.testng.Assert.assertEquals(Assert.java:399)
	at main.testcase.Wallet.withdraw_fail.w_al(withdraw_fail.java:85)
</failure>
    <system-out>{&quot;systemVersion&quot;:&quot;1&quot;,&quot;appVersion&quot;:&quot;1&quot;,&quot;clientIP&quot;:&quot;192.168.9.55&quot;,&quot;deviceNumber&quot;:&quot;a750a6d6-edf1-4704-9146-bc7eed651d65&quot;,&quot;sessionGuid&quot;:&quot;b43c796e-1b43-45e8-a2bb-d84dd01ad034&quot;,&quot;token&quot;:&quot;ccefa4c6-4508-4dce-a453-170b35719942&quot;,&quot;shopId&quot;:&quot;11c8f677-9fe0-4fc8-b895-76e4b86e5dd9&quot;,&quot;account&quot;:&quot;汤涛&quot;,&quot;bankNo&quot;:&quot;123123456632123000&quot;,&quot;bankName&quot;:&quot;中国工商银行&quot;,&quot;branchName&quot;:&quot;杭州滨江支行杭州滨江支行杭州滨江支行杭州滨江支行杭州滨江支行杭州滨江支行杭州滨江支行杭州滨江支行杭州滨江支行&quot;,&quot;amount&quot;:1.52,&quot;password&quot;:&quot;32005beb6cc4314fe07aa6b0c9208f99&quot;}
{
	&quot;code&quot;:10509,
	&quot;data&quot;:null,
	&quot;message&quot;:&quot;用户未认证，无法进行提现,请认证后再进行提现！&quot;,
	&quot;success&quot;:false
}
</system-out>
  </testcase>
  <testcase time="2.774" classname="main.testcase.Wallet.withdraw_already" name="w_al"/>
  <testcase time="0.052" classname="main.testcase.Discover.discoveryModules" name="dc_code"/>
  <testcase time="0.044" classname="main.testcase.Discover.discoveryModules" name="dc_code"/>
  <testcase time="0.041" classname="main.testcase.Discover.discoveryModules" name="dc_code"/>
  <testcase time="0.035" classname="main.testcase.ManagementClerk.queryAssistantInformation" name="QAI_code"/>
  <testcase time="0.022" classname="main.testcase.ManagementClerk.queryAssistantInformation" name="QAI_code"/>
  <testcase time="0.028" classname="main.testcase.ManagementClerk.queryAssistantInformation" name="QAI_code">
    <failure message="expected object to not be null" type="java.lang.AssertionError">java.lang.AssertionError: expected object to not be null
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.assertNotNull(Assert.java:423)
	at org.testng.Assert.assertNotNull(Assert.java:408)
	at main.testcase.ManagementClerk.queryAssistantInformation.QAI_code(queryAssistantInformation.java:63)
</failure>
    <system-out>http://192.168.1.16:80/api/ucenter/employee/helper?employeeMobile=15658893333&amp;sessionGuid=b43c796e-1b43-45e8-a2bb-d84dd01ad034&amp;shopId=11c8f677-9fe0-4fc8-b895-76e4b86e5dd9&amp;systemVersion=1&amp;clientIP=192.168.9.55&amp;deviceNumber=c812c5cf-e48d-4dc1-9ee3-e3b4e99fa2b1&amp;token=ccefa4c6-4508-4dce-a453-170b35719942&amp;appVersion=1
{
	&quot;code&quot;:200,
	&quot;data&quot;:null,
	&quot;message&quot;:&quot;查询成功&quot;,
	&quot;success&quot;:true
}
</system-out>
  </testcase>
  <testcase time="0.024" classname="main.testcase.ManagementClerk.queryAssistantInformation" name="QAI_code"/>
  <testcase time="0.027" classname="main.testcase.ManagementClerk.queryAssistantInformation" name="QAI_code"/>
  <testcase time="0.231" classname="main.testcase.ManagementClerk.shopAssistantList" name="SAL_code"/>
  <testcase time="0.029" classname="main.testcase.ManagementClerk.shopAssistantList" name="SAL_code"/>
  <testcase time="0.024" classname="main.testcase.ManagementClerk.shopAssistantList" name="SAL_code"/>
  <testcase time="0.034" classname="main.testcase.ManagementClerk.shopAssistantList" name="SAL_code"/>
  <testcase time="0.214" classname="main.testcase.ManagementClerk.shopAssistantList" name="SAL_code"/>
  <testcase time="0.026" classname="main.testcase.ManagementClerk.shopAssistantList" name="SAL_code"/>
  <testcase time="0.217" classname="main.testcase.ManagementClerk.shopAssistantList" name="SAL_code"/>
  <testcase time="0.219" classname="main.testcase.ManagementClerk.shopAssistantList" name="SAL_code"/>
  <testcase time="0.779" classname="main.testcase.ManagementClerk.addShopAssistant" name="asa">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:179)
	at main.testcase.ManagementClerk.addShopAssistant.asa(addShopAssistant.java:81)
</failure>
    <system-out>{&quot;systemVersion&quot;:&quot;1&quot;,&quot;appVersion&quot;:&quot;1&quot;,&quot;clientIP&quot;:&quot;192.168.9.55&quot;,&quot;deviceNumber&quot;:&quot;a750a6d6-edf1-4704-9146-bc7eed651d65&quot;,&quot;sessionGuid&quot;:&quot;b43c796e-1b43-45e8-a2bb-d84dd01ad034&quot;,&quot;token&quot;:&quot;ccefa4c6-4508-4dce-a453-170b35719942&quot;,&quot;shopId&quot;:&quot;11c8f677-9fe0-4fc8-b895-76e4b86e5dd9&quot;,&quot;employeeName&quot;:&quot;小汤哥5&quot;,&quot;employeeUsername&quot;:&quot;15658890631&quot;}
{
	&quot;code&quot;:200,
	&quot;data&quot;:null,
	&quot;message&quot;:&quot;实名认证、店铺信息已认证，才可操作。&quot;,
	&quot;success&quot;:false
}
</system-out>
  </testcase>
  <testcase time="0.796" classname="main.testcase.ManagementClerk.addShopAssistant" name="asa"/>
  <testcase time="0.784" classname="main.testcase.ManagementClerk.addShopAssistant" name="asa"/>
  <testcase time="0.813" classname="main.testcase.ManagementClerk.addShopAssistant" name="asa"/>
  <testcase time="0.664" classname="main.testcase.ManagementClerk.addShopAssistant" name="asa"/>
  <testcase time="0.792" classname="main.testcase.ManagementClerk.addShopAssistant" name="asa"/>
  <testcase time="0.812" classname="main.testcase.ManagementClerk.addShopAssistant" name="asa"/>
  <testcase time="0.796" classname="main.testcase.ManagementClerk.addShopAssistant" name="asa"/>
  <testcase time="0.81" classname="main.testcase.ManagementClerk.addShopAssistant" name="asa"/>
</testsuite>